lib/excision/workers/train_classifier.ex

```diff
@@ -113,15 +110,18 @@ defmodule Excision.Workers.TrainClassifier do
   end
 
   defp load_model_and_tokenizer(model_name, num_labels, sequence_length) do
+    # TODO surface errors to user better
```

Comment:
(1) Can we use `Application.get_env(...)` and read from environment in `runtime.exs`?
(2) Does this still work if `HUGGING_FACE_API_KEY` does not exist?

Reply:
done, yes works if key doesn't exist